discovery/gossiper_test.go

```diff
@@ -3353,13 +3353,23 @@ func TestPropagateChanPolicyUpdate(t *testing.T) {
 
 	const numChannels = 3
 	channelsToAnnounce := make([]*annBatch, 0, numChannels)
+	channelPoints := make([]OptionalMsgField, 0, numChannels)
```

Comment:
Alternatively, you could make unique channel points within `createAnnouncements` the same way without having to use the functional options.